testhelpers/steps.go

```diff
@@ -67,6 +67,10 @@ func (ctx *TestContext) SetupTestContext(data interface{}) { // nolint
 	// reset the seed to get predictable results on PRNG for tests
 	rand.Seed(1)
 
+	// fix the current time
+	currentTime := time.Now()
+	monkey.Patch(time.Now, func() time.Time { return currentTime })
```

Comment:
It's hard to predict where it is not needed. I don't see any disadvantages, so, yes, I think it's a good idea to freeze time during each scenario, otherwise we would have to add this step into each scenario explicitly.

Reply:
Anyway, you'll have the DB at a different time. As it is not useful in 99% of the tests, I would implement it as a step used in some specific tests. I think you may not detect some bug because of that.

Quick fix: "// fix the current time" -> "// freeze the current time"

Reply:
We never compare DB time with the program’s time. I can’t imagine a bug that can be missed because of this.

Reply:
It was two different remarks.